core/src/main/java/dk/alexandra/fresco/lib/lp/LPSolver.java

```diff
@@ -154,9 +153,8 @@ private boolean checkDimensions(LPTableau tableau, Matrix<DRes<SInt>> updateMatr
    */
   private DRes<LPState> phaseTwoProtocol(ProtocolBuilderNumeric builder, LPState state) {
     return builder.seq((seq) -> seq.seq(
-        new ExitingVariable(state.tableau, state.updateMatrix, state.enteringIndex, state.basis))
-    ).pairInPar(
-        (seq, exitingVariable) -> {
+        new ExitingVariable(state.tableau, state.updateMatrix, state.enteringIndex, state.basis)))
+        .pairInPar((seq, exitingVariable) -> {
```

Comment:
Strange realignment - why did you move this?

Reply:
All the realignments because I ran auto-format on those files.